src/react/forms.jsx

```diff
@@ -5,9 +5,28 @@
 
 'use strict';
 
+var formControlClass = 'mui-form-control',
+    formGroupClass = 'mui-form-group',
+    floatingLabelBaseClass = 'mui-form-floating-label',
+    floatingLabelActiveClass = floatingLabelBaseClass + '-active',
+    animationName = 'mui-form-floating-label-inserted',
+    _supportsPointerEvents;
 
-var jqLite = require('../js/lib/jqLite.js'),
-    forms = require('../js/forms.js');
+var React = require('react');
```

Comment:
Can we rely on the external React library? Otherwise Browserify will inline the React source in mui-react.js.